src/components/People/People.tsx

```diff
@@ -0,0 +1,90 @@
+import { Link, useParams } from 'react-router-dom';
+import data from '../../../public/api/people.json';
```

Comment:
Ensure that the path to `people.json` is correct relative to the file structure. If the file is not found, it could lead to runtime errors.